src/scripts.js

```diff
@@ -35,6 +36,16 @@ function whateveriwant() {
    cookBook = new RecipeRepository(recipesInfo,  ingredientsInfo)
   // console.log(cookBook.recipeData);
   multipleButtons();
+  makeRecipeCard();
+}
+
+function makeRecipeCard() {
+  cookBook.recipeData.forEach(recipe => {
+    recipeCard.innerHTML += 
+    `<img class="food-preview" src=${recipe.image}>
+      <h2>${recipe.name}</h2>`
```

Comment:
If there is time, let's add a box around each card to make them feel more like cards